apps/gql/features/auth/index.ts

```diff
@@ -0,0 +1,20 @@
+import { type Session } from "@kampus/next-auth";
+
+import { env } from "../../env";
+
+export const getSession = async (request: Request): Promise<Session | null> => {
```

Comment:
Do we need the return type here? we have proper return statements in `try .. catch` block.

Reply:
I was trying to be explicity but not needed so I'll remove.